src/coordinatorapi.cpp

```diff
@@ -409,6 +409,31 @@ void CoordinatorAPI::updateCoordCkptDir(const char *dir)
   _coordinatorSocket.writeAll(dir, strlen(dir) + 1);
 }
 
```

Comment:
Suggest adding a comment to introduce `updateGlobalCkptDir()`.  Something like:
> //  Invoked by a plugin via `dmtcp.h:dmtcp_set_global_ckpt_dir()` and
// via `dmtcpplugin.cpp:dmtcp_set_global_ckpt_dir()`